db/sqlalchemy.py

```diff
@@ -15,8 +15,13 @@ def __init__(
 
     def make_declarative_base(self, model, metadata=None):
         """Creates or extends the declarative base."""
-        if self.Model is None:
-            self.Model = super().make_declarative_base(model, metadata)
+        try:
+            if self.Model is None:
+                self.Model = super().make_declarative_base(model, metadata)
 
-        set_query_property(self.Model, self.session)
-        return self.Model
+            set_query_property(self.Model, self.session)
```

Comment:
Why are you setting this property?

Reply:
This property was already defined before isn't?

Reply:
Sorry - my bad. You're correct.